doc/book/service-manager.md

```diff
@@ -44,22 +44,31 @@ return [
     'log' => [
         'MyLogger' => [
             'writers' => [
-                [
+                'stream' => [
                     'name' => 'stream',
-                    'priority' => Logger::DEBUG,
+                    'priority' => 1,
```

Comment:
Please use the correct constant with namespace here.

Reply:
Ok, but it should be indicated that this priority is only a priority in the queue and it is inversed: so "1" (alert, default for the logger) will be triggered after the "7" (debug).

Reply:
@Daniel-KM 
Please do not forget the topic of this chapter "Service Manager Integration". So this is the wrong place to explain this.